CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+We are always happy about support for our project. If you dicide to help us spreading awareness for privacy, just start with the following steps: 
+* fork the project,
+* create a new branch, 
+* commit your changes 
+* and open a pull request to `master` 
+  
+(More in ["Branching & Naming"](#branching--naming))
+
+For your Git Commit Messages, please orientate on the guidance in the following [article](https://chris.beams.io/posts/git-commit/):
+
+> - Limit the subject line to 50 characters
+> - Capitalize the subject line
+> - Use the imperative mood in the subject line
+> - Use the body to explain what and why, less how
+
+### Pull Requests
+
+Every new feature must be developed on a feature branch and merged into master. **Please do not push directly to master!** We also provide a [Pull Request Template]([/.github/pull_request_template.md](https://github.com/codeuniversity/smag-mvp/blob/master/.github/pull_request_template.md)) for additional guidance. In any case, the Pull Request has to be reviewed and approved by at least one other developer before merging. Please make sure to [reference the associated issue(s)](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) in the pull request.
```

Comment:
PR-Template link is broken
```suggestion
Every new feature must be developed on a feature branch and merged into master. **Please do not push directly to master!** We also provide a [Pull Request Template]( [.github/pull_request_template.md](.github/pull_request_template.md) ) for additional guidance. In any case, the Pull Request has to be reviewed and approved by at least one other developer before merging. Please make sure to [reference the associated issue(s)](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) in the pull request.
```

Reply:
I put a fixed link in there again to have it working in the pdf